go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -152,11 +140,9 @@ func (npw *nodePortWatcher) updateServiceFlowCache(service *kapi.Service, add bo
 					npw.ofm.updateFlowCacheEntry(key, nodeportFlows)
 				} else {
 					npw.ofm.updateFlowCacheEntry(key, []string{
-						fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, tp_dst=%d, "+
-							"actions=%s",
-							cookie, npw.ofportPhys, flowProtocol, svcPort.NodePort, actions),
-						fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, tp_src=%d, "+
-							"actions=output:%s",
+						fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, tp_dst=%d, actions=%s",
```

Comment:
nit: to keep things consistent maybe use "actions=output:%s", the `output:` prefix is used in other places in this file when adding OF rule which outputs a packet on a certain port ?

Reply:
Unreverted the `disable mtu` flag change.